src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -67,7 +75,8 @@ public void onCompletion(RecordMetadata md, Exception e) {
 public class MaxwellKafkaProducer extends AbstractProducer {
 	static final Object KAFKA_DEFAULTS[] = {
 		"compression.type", "gzip",
-		"metadata.fetch.timeout.ms", 5000
+		"metadata.fetch.timeout.ms", 5000,
+		"retries", 1
```

Comment:
Why retries = 1? If your delivery ever fails, then Maxwill drop the message and keep going on. Except it won't checkpoint binlog offsets. So how will you ever get back to the dropped message? By crashing/restarting?